docs(approvals): add documentation of approvals (#26)
ForestAdmin/documentation-user-guide: other-tabs/collaboration/approval-requests.md

# Approval requests

{% hint style="info" %}
You must be on a [Forest Admin Pro plan](https://www.forestadmin.com/pricing) to have access to this feature.
{% endhint %}

> Approval requests are created by smarts actions that require manual validation to be executed according to the [Approval workflow](../../project-settings/teams-and-users/manage-roles.md#approval-workflow-permissions). On this screen, you'll find all the actions linked to these approvals, divided into three lists: &#x20;

Reviewer: ```suggestion
Approval requests are created by triggering smart actions that require manual validation according to the [Approval workflow](../../project-settings/teams-and-users/manage-roles.md#approval-workflow-permissions). On this screen, you will find all approval requests divided into three lists: &#x20;
```

> - __Created__:  This is the list of approvals requests you've created using acton smarts.

Reviewer: ```suggestion
- __Created__:  This is the list of pending approvals you have generated.
```

> - __To review__: This is the list of approval requests you have to approve or reject.

Reviewer: ```suggestion
- __To review__: This is the list of approval requests you have the right to approve or reject based on role permission.
```

> - __History__: This is the history of approvals requests that have been processed and closed.

Reviewer: ```suggestion
- __History__: This is the history of approval requests that have been processed and closed.
```

![](../../.gitbook/assets/approvals-global-screen.png)

## Action on approval requests

> You can open every approval request you have to process and choose to validate or reject them on this modal:

Reviewer: ```suggestion
By clicking on an approval, a modal will open. In there you will find the execution context of the action (the targeted record and any form value that was defined on the action). After review, you can either validate or reject the approval:
```
